setup.py

```diff
@@ -21,18 +24,25 @@
 with open('README.rst') as f:
     readme = f.read()
 
+PY3 = (sys.version_info[0] >= 3)
+
+install_requires = [
+    "pyOpenSSL >= 0.13",
+    "requests >= 2.1.0",
+    "service_identity >= 14.0.0",
+    "six"
+]
+
+if PY3:
```

Comment:
Well, by default Wheels are specific to either Python 2 or Python 3, however in the ``setup.cfg`` this is stating the wheel is universal (e.g. the same build will work on both Python 2 and Python 3). You have a few options here depending on what versions of things you want to support and how efficient you want to be.

1. Remove the ``universal=1`` from ``setup.cfg`` and have different wheels for Python 2 and Python 3. *Note: This will only work as long as your build process only differs by 2.x vs 3.x*.
2. Continue to use ``universal=1`` in the ``setup.cfg``, keep the programatic conditional logic in ``setup.py``, and duplicate all of the dependency information into it. When you do this the wheel project, when building the wheel, will use that instead of ``install_requires``. This will allow you to build a universal wheel that works and have a ``setup.py`` that works with all versions of pip and setuptools you can reasonably (and unreasonably tbh) find.
3. Continue to use ``universal=1`` in the ``setup.cfg`` but remove the programatic conditional logic from ``setup.py`` and instead rely on the environment marker support. This gets you a single universal wheel without duplicating the dependency information, however it only works on setuptools >= 0.7 and only pip 6.0+ when install from a sdist (pip 1.4+ when installing from a wheel, wheels were opt in for 1.4, opt out for 1.5).

So essentially you get to choose which option hits the right level of compatibility, efficiency, and duplication for your individual project. It's intended that the third option be the right option in the long run, but you'll have to decide if the compatibility concerns are too great for treq.